docs/user/alerting/alerting-getting-started.asciidoc

```diff
@@ -157,7 +157,7 @@ Pre-packaged *alert types* simplify setup, hide the details complex domain-speci
 
 If you are using an *on-premises* Elastic Stack deployment:
 
-* In the kibana.yml configuration file, add the <<alert-action-settings-kb,`xpack.encryptedSavedObjects.encryptionKey`>> setting.
+* In the kibana.yml configuration file, add the <<action-settings,`xpack.encryptedSavedObjects.encryptionKey`>> setting.
```

Comment:
Should this link to the "General settings" section instead of the "Action settings" section?